python/aswfdocker/data/versions.yaml

```diff
@@ -405,9 +405,9 @@ ci-images:
     - "2020.8"
     - "2021.6"
   opencue:
-    - "2019"
-    - "2020"
-    - "2021"
+    - "2019.10"
+    - "2020.10"
+    - "2021.10"
     - "2021.7"
```

Comment:
I had noticed these had no subversion but thought it was intentional - should have asked sorry! Did you mean to leave 2021.7 here?

Reply:
Oh thanks! I actually used a vscode extension to increment the version numbers by 0.1 and it worked... `2018.9+0.1=2019`...
Thanks I fixed it properly now!